src/github.js

```diff
@@ -156,12 +161,25 @@ const listRepoContents = async (token, repo, path = '') => {
  * @returns {string[]} - List of repos. Example: ["silinternational/vulnerability-scanner"]
  */
 module.exports.listRepos = async (org, token) => {
-  const response = await request("GET /orgs/:org/repos", {
-    headers: {
-      authorization: "token " + token,
-    },
-    org: org,
-  })
-  const repos = response.data
-  return repos.map(repo => repo.full_name)
+  const allRepos = []
+  const pageSize = 100
+  
+  let page = 0
+  let thereAreMore = true
+  while (thereAreMore) {
+    page++
+    const response = await request("GET /orgs/:org/repos?per_page=:pageSize&page=:page", {
+      headers: {
+        authorization: "token " + token,
+      },
+      org: org,
+      page: page,
+      pageSize: pageSize
+    })
+    const repos = response.data
+    allRepos.push(...repos)
+    thereAreMore = (repos.length === pageSize)
```

Comment:
Does this work correctly if the total number of repos is a multiple of the page size? In other words, what kind of response do you get if you request a page after the last? I took a quick skim of https://developer.github.com/v3/guides/traversing-with-pagination/ and it doesn't answer the question, but refers to `rels` to determine whether you're on the last page.

Reply:
It does seem to work correctly, yes. To verify, I tried setting the page size to `1`, and I ended up with the same total number of results and no error messages.